### ISSUE TITLE
Exceptions reported on many java files 

### ISSUE THREAD

**Issue by amaembo** [johnnyleitrim/IntelliJParameterOrderInspection#4]
E.g. when I open [this file](https://github.com/JetBrains/intellij-community/blob/master/jps/jps-builders/src/org/jetbrains/jps/cmdline/LogSetup.java) from IntelliJ community project

```
In file: .../jps/jps-builders/src/org/jetbrains/jps/cmdline/LogSetup.java

java.lang.IndexOutOfBoundsException: Index 2 out of bounds for length 2
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
	at java.base/java.util.Objects.checkIndex(Objects.java:372)
	at java.base/java.util.ArrayList.get(ArrayList.java:459)
	at com.jfarrelly.intellij.plugin.code.inspection.MethodParameterOrderVisitor.checkParameterPosition(MethodParameterOrderVisitor.java:61)
	at com.jfarrelly.intellij.plugin.code.inspection.MethodParameterOrderVisitor.visitMethodCallExpression(MethodParameterOrderVisitor.java:52)
	at com.intellij.psi.impl.source.tree.java.PsiMethodCallExpressionImpl.accept(PsiMethodCallExpressionImpl.java:125)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
```

**Comment by johnnyleitrim** [comment]
Hi @amaembo - This should be fixed in version 1.0.1 of the plugin.

See https://github.com/johnnyleitrim/IntelliJParameterOrderInspection/pull/3 for more details.

**Comment by johnnyleitrim** [comment]
I downloaded the IntelliJ community edition code, and I can confirm that the exception no longer happens with v1.0.1
